### DISCUSSION TITLE
Enhancement: Fix Arrow Misplacement

### DISCUSSION THREAD

**Discussion by superguyoffa** [FunkinCrew/Funkin#1590] [Ideas]
I noticed just today that the left set of arrows are closer to the left side of the screen than the right set of arrows. I cannot unsee it now, pls fix.

edit: week 6 has the same issue, now I have no idea if this is intentional or not.

![bruhh](https://user-images.githubusercontent.com/78882471/107578430-05e4b900-6bf4-11eb-8da9-34e481df99af.png)

**Comment by shecravesaldo** [comment]
Weird. Kind of wished I didn't see this because now i cant unsee it either lml

**Comment by RagdollClash** [comment]
Duplicate of #223

**Comment by superguyoffa** [comment]
> Duplicate of #223

oh alright, i didn't realize someone already posted this as an issue.
